config.go

```diff
@@ -2,35 +2,57 @@ package main
 
 import (
 	"flag"
+	"os"
 
 	"github.com/vharitonsky/iniflags"
 )
 
 var (
+	VERSION    = "1.6.0"
 	appVersion = "unknown"
 	buildTime  = "unknown"
 )
 
 var (
-	logFile           = flag.String("logfile", "/var/log/smtprelay.log", "Path to logfile")
-	hostName          = flag.String("hostname", "localhost.localdomain", "Server hostname")
-	welcomeMsg        = flag.String("welcome_msg", "", "Welcome message for SMTP session")
-	listen            = flag.String("listen", "127.0.0.1:25 [::1]:25", "Address and port to listen for incoming SMTP")
-	localCert         = flag.String("local_cert", "", "SSL certificate for STARTTLS/TLS")
-	localKey          = flag.String("local_key", "", "SSL private key for STARTTLS/TLS")
-	localForceTLS     = flag.Bool("local_forcetls", false, "Force STARTTLS (needs local_cert and local_key)")
-	allowedNets       = flag.String("allowed_nets", "127.0.0.1/8 ::1/128", "Networks allowed to send mails")
-	allowedSender     = flag.String("allowed_sender", "", "Regular expression for valid FROM EMail addresses")
+	logFile       = flag.String("logfile", "/dev/stdout", "Path to logfile")
+	hostName      = flag.String("hostname", "localhost.localdomain", "Server hostname")
+	welcomeMsg    = flag.String("welcome_msg", "", "Welcome message for SMTP session")
+	listen        = flag.String("listen", "127.0.0.1:25 [::1]:25", "Address and port to listen for incoming SMTP")
+	metricsListen = flag.String("metrics_listen", ":8080", "Address and port to listen for metrics exposition")
```

Comment:
This should be disabled per default. With :8080 it listens on all available interfaces which might be safe in a docker environment but not on real servers with public interfaces!